core/util.go

```diff
@@ -422,3 +423,14 @@ func RetryBackoff(retries int, base, max time.Duration, factor float64) time.Dur
 	backoff *= (1 - retryJitter) + 2*retryJitter*mrand.Float64()
 	return time.Duration(backoff)
 }
+
+// HashNames returns the sha256 hash of the lowercased, comma joined
```

Comment:
(You might want to refresh, I think you are looking at a slightly outdated diff at the moment)